CHANGELOG.md

```diff
@@ -1,458 +1,11 @@
 # Changelog
 
-## v2.2.0 (2021-04-23)
+## v3.0.0 (2021-05-14)
```

Comment:
Do we want to keep the history for those that want to see where this came from? Especially since we auto-gen the changelog (though that can be configured)?

Reply:
depends a bit on where we are going to publish this. npm package `node-saml` is currently taken by my previous fork/strip down of passport-saml but I'm happy to release that name. As that one is already at 3.0 we'd have to go to 4.0 for the first release from this repo. I thought that the old changelog is not needed here and have therefore referenced the passport-saml changelog for anything prior to what is going to be released from this repo.

Or we have to use al alternate npm package name, e.g. `@node-saml/node-saml`

Reply:
I was expecting that we'd release a pair of 4.0.0 when we do the separation and then we could publish over the `node-saml` package name like you suggest.

Reply:
I found a problem with [gren](https://github.com/github-tools/github-release-notes) which is affecting this repo. We'll have to revisit change logs when [this](https://github.com/github-tools/github-release-notes/issues/116) gets fixed.